lighthouse-core/report/html/renderer/pwa-category-renderer.js

```diff
@@ -46,14 +46,53 @@ class PwaCategoryRenderer extends CategoryRenderer {
     return categoryElem;
   }
 
+  /**
+   * @param {LH.ReportResult.Category} category
+   * @return {DocumentFragment}
+   */
+  renderScoreGauge(category) {
+    // Defer to parent-gauge style if category error.
+    if (category.score === null) {
+      return super.renderScoreGauge(category);
+    }
+
+    const tmpl = this.dom.cloneTemplate('#tmpl-lh-gauge--pwa', this.templateContext);
+    const wrapper = /** @type {HTMLAnchorElement} */ (this.dom.find('.lh-gauge--pwa__wrapper',
+      tmpl));
+    wrapper.href = `#${category.id}`;
```

Comment:
I'm wondering if we should also clarify the state in a tooltip.

We could do with a full lh-tooltip and bulleted list, but here's a minimal `[title]` example: 
![image](https://user-images.githubusercontent.com/39191/48652605-39a0ec00-e9b5-11e8-8931-04f32eec814a.png)

Regardless, I don't think it should hold up this PR. (which is indeed v clean)

Reply:
yeah, it feels like it should have a tooltip. I'll add it to the list

Reply:
Alternative idea: 
Fast and reliable: 3/3, Installable: 3/3, PWA Optimized: 4/6